mozilla/blurts-server
# MNTOR-1130: Animations preview

# References: 
Jira: [MNTOR-1130](https://mozilla-hub.atlassian.net/browse/MNTOR-1130)



# Description

Adds a test preview for Lottie animations.

# Screenshot

<img width="631" alt="mntor-1130-lottie-preview" src="https://user-images.githubusercontent.com/13835474/232504333-905005a2-b7db-41e8-9e11-fc93f9b07ec4.png">

# How to test

1. Navigate to `/preview/animations` — the `/preview` route should be only available for `dev` and `staging`
2. Select and play/pause animations

# Checklist (Definition of Done)
